src/rage/RageVector3.hpp

```diff
@@ -69,12 +66,27 @@ inline Vector3 operator/(Vector3 lhs, float rhs)
 	return lhs;
 }
 
-inline Vector3 operator*(Vector3 lhs, Vector3 const &rhs)
+// Little known fact: There are two ways to multiply two vectors:
+// Cross product, and dot product.  Because they are both vector
+// multiplication, neither of them should use the * operator.
+// And combining with an assignment operator is more wrong because it
+// forces more operations because the person doing the multiplication
+// always wants to put the result in a different vector. -Kyz
+inline Vector3 CrossProduct(Vector3 const& lhs, Vector3 const & rhs)
```

Comment:
This was the only one that got in my way.  I needed a working cross product function for the NewField and didn't want to touch functions I didn't have to.